frontend/src/layout/__stories__/Navigation.stories.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import { Meta } from '@storybook/react'
+import { Layout } from 'antd'
+import { keaStory } from 'lib/storybook/kea-story'
+import { MainNavigation } from '../navigation'
+
+import { TopNavigation } from '../navigation/TopNavigation'
+
+import navigationInitial from './navigation.initial.json'
+
+export default {
+    title: 'PostHog/Components/TopNavigation',
+    parameters: {
+        layout: 'fullscreen',
+    },
+} as Meta
+
+export const PlainTopNavigation = keaStory(
+    () => (
+        <Layout>
+            <MainNavigation />
+            <Layout>
+                <TopNavigation />
```

Comment:
Somehow the top-right part of the navigation disappears:
![2021-09-28 09 36 59](https://user-images.githubusercontent.com/53387/135043906-0a67ff53-b91c-49ce-86c3-a86de1934a64.gif)

Reply:
That's because of the issue discussed in https://github.com/PostHog/posthog/pull/6027#issuecomment-924856640
This looks like it could somehow help, but I've no idea how to make use of it:
<img width="223" alt="Screen Shot 2021-09-29 at 15 23 55" src="https://user-images.githubusercontent.com/4550621/135290763-37676061-e98f-4ef4-a05e-fec8ca34ff14.png">